src/main/java/com/wso2telco/kafka/MessageSender.java

```diff
@@ -10,16 +12,22 @@
 
 import static com.wso2telco.util.CommonConstant.AXP_ANALYTICS_LOGGER;
 import static com.wso2telco.util.CommonConstant.kafkaEnabled;
-
+import static com.wso2telco.util.Constants.*;
 public class MessageSender {
+    private static TimeOutCount errorCount=new TimeOutCount();
+    private static boolean runtimeKafkaEnabled=true;
+    private static long runtimeKafkaUpdateMillis=Long.MAX_VALUE;
 
     public void sendMessage(String transactionLog) {
+        if(System.currentTimeMillis() - runtimeKafkaUpdateMillis >=RUNTIMEKAFKA_FRESHNESS_THRESHOLD)
```

Comment:
test this behavior to check when MessageSender object initiate runtimeKafkaEnabled=true always?